diff -urN ../resources-original/w3c/TR/xmldsig-core2/xmldsig2-schema.xsd w3c/TR/xmldsig-core2/xmldsig2-schema.xsd
--- ../resources-original/w3c/TR/xmldsig-core2/xmldsig2-schema.xsd	2019-04-30 16:33:49.253571000 +0200
+++ w3c/TR/xmldsig-core2/xmldsig2-schema.xsd	2019-04-30 16:34:42.581584800 +0200
@@ -1,3 +1,4 @@
+<!--
 #
 # Copyright ©[2011] World Wide Web Consortium 
 # (Massachusetts Institute of Technology,  
@@ -9,7 +10,7 @@
 # PURPOSE. 
 # [1] http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231
 #
-
+-->
 <?xml version="1.0" encoding="utf-8"?>
 <schema xmlns="http://www.w3.org/2001/XMLSchema" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:dsig11="http://www.w3.org/2009/xmldsig11#" xmlns:dsig2="http://www.w3.org/2010/xmldsig2#" targetNamespace="http://www.w3.org/2010/xmldsig2#" elementFormDefault="qualified" version="0.1">
 	<import namespace="http://www.w3.org/2000/09/xmldsig#" schemaLocation="xmldsig-core-schema.xsd"/>
